Client/Frontend/Browser/BrowserViewController/BrowserViewController+Onboarding.swift

```diff
@@ -42,9 +42,8 @@ extension BrowserViewController {
             // 60 days has passed since the user last saw the onboarding.. it's time to show the onboarding again..
             if daysUntilNextPrompt <= Date() && !isRewardsEnabled {
                 guard let onboarding = OnboardingNavigationController(
-                    profile: profile,
-                    onboardingType: .existingUserRewardsOff(currentProgress),
-                    rewards: rewards
+                    browser: self,
+                    onboardingType: .existingUserRewardsOff(currentProgress)
```

Comment:
These changes (passing BVC into the onboarding flow) are fine for now but I don't want to see them when we implement the new onboarding flow